src/main/java/com/contrastsecurity/maven/plugin/InstallAgentContrastMavenMojo.java

```diff
@@ -25,7 +25,19 @@ public void execute() throws MojoExecutionException {
 
         getLog().info("Agent downloaded.");
 
-        project.getProperties().setProperty("argLine", buildArgLine(project.getProperties().getProperty("argLine")));
+        String applicationName;
+        if (StringUtils.isNotBlank(appId)) {
+            applicationName = getAppName(contrast, appId);
+            retrievedAppName = applicationName;
```

Comment:
I don't see `retrievedAppName` used anywhere

Reply:
Oh now I see it declared at the top of the file. Regardless, having `retrievedAppName` `applicationName` and `appName` all running around makes things difficult to follow.

Reply:
@gmoore I removed the `retrievedAppName`, so it should be a little cleaner now.